Sockets/BackEndMonitoringSockets/include/CAcceptorWrapper.h

```diff
@@ -1,30 +1,39 @@
 #pragma once
+#include "CServiceHandler.h"
 #include "CAcceptor.h"
-#include "DataReceiver.h"
-#include "CServiceConnectionHandler.h"
+#include "CSocketWrapper.h"
 #include "CThreadPool.h"
-#include "CEvent.h"
+
+class DataReceiver;
+class CEvent;
+class CSocketWrapper;
 
 // This class works with server 
 class CAcceptorWrapper
 {
 public:
-	CAcceptorWrapper(int port, const std::string& ip_address, 
-		CEvent& event, std::shared_ptr<CThreadPool> m_pool,
-		std::shared_ptr<CLogger> logger);
+	CAcceptorWrapper(int port, const std::string& ip_address, CEvent& event, 
```

Comment:
CAcceptorWrapper should have the next public methods:
CAcceptorWrapper(const SAcceptorWrapParams &);
bool Inittialize();// InitializeWSA , Create CAcceptor object and initialize it.
bool Execute():// this method should run in other thread and checking global stop event, create a new connection using CAcceptor and enque it in the thread pool.
Maybe setter for thread poll will be need.